README.md

```diff
@@ -4,7 +4,7 @@ Wrapper of streadway/amqp that provides reconnection logic and sane defaults. Hi
 
 Supported by [Qvault](https://qvault.io)
 
-[![](https://godoc.org/github.com/wagslane/go-rabbitmq?status.svg)](https://godoc.org/github.com/wagslane/go-rabbitmq)![Deploy](https://github.com/wagslane/go-rabbitmq/workflows/Tests/badge.svg)
+[![](https://godoc.org/github.com/claranet/go-rabbitmq?status.svg)](https://godoc.org/github.com/claranet/go-rabbitmq)![Deploy](https://github.com/claranet/go-rabbitmq/workflows/Tests/badge.svg)
```

Comment:
Was this added by accident? We aren't changing the import path on the main repo

Reply:
> Hi @wagslane Oh yes, I mixed things up while providing the PR but also make it available while the changes are not in your repository. I will clean this up and provide the PR again. 
Same here